### ISSUE TITLE
[QA] Broken gameplay and no NPC animation on Teleperformance

### ISSUE THREAD

**Issue by anicalbano** [decentraland/unity-explorer#1383]
### **Explorer Alpha build version:**
v0.11.0

### **Issue Description:**

The climbing wall game can be started, however, there is an asset on the floor that lifts up the player and it looks like the NPC is starting the climb through the floor up. 
Said NPC is in a static position, not performing the correct movements of the climbing.

### **Steps to reproduce:**

  1. Start the climbing wall game selecting an NPC
  2. Use the W to climb untill the NPC is visible

### **Expected behaviour:**
The game must be played normally, with the NPC visible as soon as it starts

### **Current behaviour:**
The NPC appears through the floor when mantaining the W pressed. There is no movement on it, it stays on the same position (has no animation). 

#### **Evidence:**

https://github.com/decentraland/unity-explorer/assets/139157886/9edcc9af-d2c3-47bc-ba47-a54af902da2d

**Comment by lorux0** [comment]
Important notes about the investigation of this issue.
The climbing character had multiple states, but when set to `playing: true`, the `weight:0`, causing the animation to be invisible.

It is stated that by SDK team that:

> If the weight is explicitly set to 0 then the animation would not be viewed.

So if the animation is not visible then is the expected thing to do. Seems like unity-renderer does not consider the weight for the bone blending, but on the current animation system in the new explorer, this will work as it should be.

In conclusion the scene should fix the weight value.

**Comment by anicalbano** [comment]
Closing the issue due to what @lorux0 mentioned in the previous comment

**Comment by Kaze-No-Kai** [comment]
I get that the the scene should fix the weight value, but it is an sdk7 code that worked well in both old client and browser.
James from Vegas City mentioned that they have this instance across several scenes.
In my opinion at least, if an SDK7 scene worked in the old client, it should also work in the new client.
Thanks in advance for checking it again :pray:

**Comment by Kaze-No-Kai** [comment]
@Ludmilafantaniella the animation of the NPC is still not working, there needs to be visible how the hands/legs move like when you "climb" lets keep it open

**Comment by sandrade-dcl** [comment]
Does someone knows if this scene has been deleted or moved to another coords?

**NEW EXPLORER:**

![Image](https://github.com/user-attachments/assets/072f329d-67bb-4ddf-bc60-231264abbd12)

**OLD EXPLORER:**

![Image](https://github.com/user-attachments/assets/12006869-57f8-4ce1-8ff1-41b98168b1d7)

cc/ @Kaze-No-Kai @anicalbano @Ludmilafantaniella

**Comment by Kaze-No-Kai** [comment]
Checking with the creator

**Comment by Kaze-No-Kai** [comment]
The scene has been removed, closing this one.
